### PR TITLE
Detect log follow requests

### LINKED ISSUE #3878: Terminal: kubectl logs --follow times out
**Describe the bug**

`kubectl logs <pod> --follow`  times out with error:

```
error: unexpected EOF
```

### PR DESCRIPTION
**Pull request by jakolehm**
Fixes #3878

### CODE REVIEW

File: src/main/lens-proxy.ts [hunk 1]
```diff
@@ -40,6 +40,24 @@ export interface LensProxyFunctions {
...
 }
 
+const truthyParams = ["1", "true", ""];
+const watchParam = "watch";
+const followParam = "follow";
+
+export function isLongRunningRequest(reqUrl: string) {
+  const url = new URL(reqUrl, "http://localhost");
+
+  if (url.searchParams.has(watchParam) && truthyParams.includes(url.searchParams.get(watchParam)) ) {
```
Comment (Nokel81) [inline src/main/lens-proxy.ts]: I would use the `getBoolean` method from `src/main/routes/utils/` (and maybe move it) unless you feel that this shouldn't prioritize false.

Comment (jakolehm) [reply src/main/lens-proxy.ts]: I didn't know that function.... but that function returns false if value is empty. Is that expected behaviour also here 🤔

Comment (Nokel81) [reply src/main/lens-proxy.ts]: No it doesn't. The function `get` return `null` if the key is not set

Comment (jakolehm) [reply src/main/lens-proxy.ts]: Right, refactored to use `getBoolean`.

**Review by Nokel81 — APPROVED** [review]

**Pull request merged**